tests/unit_tests/crud/test_quality_matrix.py

```diff
@@ -122,10 +121,12 @@ def test_create_empty_entries_search():
                 "must": [
                     {"match": {"properties.ccm:replicationsource": "dummy_source"}},
                     {"term": {"path": DUMMY_UUID}},
-                    {"match": {"permissions.Read": "GROUP_EVERYONE"}},
-                    {"match": {"properties.cm:edu_metadataset": "mds_oeh"}},
-                    {"match": {"nodeRef.storeRef.protocol": "workspace"}},
-                ]
+                ],
```

Comment:
This changed, because there were no `base_filters` on one of the `Searches`, right?

Reply:
No, base filters were there, but I moved them from "must" to "filter" to remove the need to calculate score and from "match" to "term", since there is no need for a fuzzy search.
Should I add something to the documentation?